Formula/perl.rb

```diff
@@ -54,13 +53,10 @@ def install
       -Duselargefiles
       -Dusethreads
     ]
```

Comment:
Can probably add `-Dinstallstyle=lib/perl5` as alternative fix for #112150

Reply:
I was going to suggest in that PR that we see if we can avoid patching to change the `installstyle` and it appears we can.  I think this is a better approach and should be used instead.